src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball;
+        for (int i = 0; i < 3; i++) {
+            ball = new Lottery().getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
1. we should move out Lottery creation out of loop as we need only one object of it
2. variable for ball is not necessary
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```